src/index.ts

```diff
@@ -9,11 +9,9 @@ function esMultiplo(num: number, multiplo: number) {
     return false;
   }
 }
-console.log(esMultiplo(num, multiplo));
-for (let i: number = num; i < multiplo; i++) {
-  if (esMultiplo(i, multiplo)) {
-    console.log(i + " Es multiplo de " + multiplo);
-  } else {
-    console.log(i + " No es multiplo de " + multiplo);
-  }
+
+if (esMultiplo(num, multiplo)) {
+  console.log(num + " Es multiplo de " + multiplo);
+} else {
+  console.log(num + " No es multiplo de " + multiplo);
```

Comment:
Esta todo bien, solo que no entiendo porque pusiste el **if** dentro de un **for**, tampoco entiendo las condiciones del **for**.

El **for** no es necesario. Solo es necesario el if, que muestre el mensaje según los valores ingresados sean multiplos o no.